set.mm

```diff
@@ -7641,6 +7641,15 @@ an illustration of the conservativity of definitions (definitions do not
     biorfi $p |- ( ps <-> ( ps \/ ph ) ) $=
       ( wo orc wn wi orel2 ax-mp impbii ) BBADZBAEAFKBGCABHIJ $.
   $}
+  
+  $( Rewriting implication based theorems using disjunction. $)
+  
+  $( Theorem *2.26 of [WhiteheadRussell] p. 104.  See ~ pm2.27 .  (Contributed by NM,
+     3-Jan-2005.)  (Proof shortened by Wolf Lammen, 23-Nov-2012.) $)
+  pm2.26 $p |- ( -. ph \/ ( ( ph -> ps ) -> ps ) ) $=
+    ( wi pm2.27 imori ) AABCBCABDE $.
+
+  $( Contradiction and disjunction. $)
```

Comment:
Do you mean "Negation and disjunction" ?

Reply:
No, the following theorems are pm2.65 in disguise.  And this expresses contradiction.